tfhe/src/shortint/engine/server_side/mod.rs

```diff
@@ -603,9 +603,29 @@ impl ShortintEngine {
     where
         F: Fn(u64, u64) -> u64,
     {
+        // TODO this looks wrong for the Accumulator generation
```

Comment:
Should his be fixed now (this commit comes from @IceTDrinker PR) ?

Reply:
it should have been fixed by Thomas IIRC check the main branch

Reply:
In commit a0946ac509a481614d58ea6c9dc3fad70b54fc64?

Reply:
think so yes

Reply:
if you are confident with the current version launch all the test suites and we'll redo a thorough review of the new code

Reply:
All the tests (except CPU AWS Fast Tests) ran fine just before adding the last commit so I don't think rerunning them is necessary

Reply:
disagree

Reply:
they will re run anyways once we review 😄

Reply:
I will remove it

Reply:
is the change you made still required ?

Reply:
I took it from your PR
But I think so as it cleans the input if necessary instead of cleaning the result (cloned from the input)

Reply:
my PR predates the fix from Thomas, i.e. the fix from Thomas should be the one to keep